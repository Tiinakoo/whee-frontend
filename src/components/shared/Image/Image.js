import React from "react";
import styles from "./Image.module.scss";

const Image = ({ src, alt }) => (
  <div className={styles.image}>
    <img src={src} alt={alt} />
  </div>
);

export default Image;
