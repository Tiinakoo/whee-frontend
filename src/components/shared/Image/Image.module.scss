@import "../../../styles/baseline";
@import "../../../styles/mixins";

.image {
  padding: $baseline 0;

  @include md {
    padding: 2 * $baseline 4 * $baseline;
  }
}
