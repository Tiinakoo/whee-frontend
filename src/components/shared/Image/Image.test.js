import React from "react";

import { mount } from "enzyme";
import Image from "./Image";

describe("Image", () => {
  it("renders", () => {
    const component = mount(<Image />);

    expect(component).toMatchSnapshot();
  });
});
