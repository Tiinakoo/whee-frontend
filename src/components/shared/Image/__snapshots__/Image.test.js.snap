// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image renders 1`] = `
<Image>
  <div
    className="image"
  >
    <img />
  </div>
</Image>
`;
